components/events/eventCard.tsx

```diff
@@ -16,21 +16,17 @@ interface EventProps{
 export default function EventCard({date, title, imageUrl, time, description, slug}: EventProps): JSX.Element {
     
     return (
-        <div className="flex flex-col items-center border rounded-lg shadow md:flex-row md:max-w-xl max-w-md border-gray-700 bg-gray-800 hover:bg-gray-700">
+        <div className='flex flex-col items-stretch text-start m-[1em]  justify-stretch min-w-[302px] border rounded-lg shadow md:flex-row md:max-w-xl max-w-md border-gray-700 bg-gray-800 hover:bg-gray-700'>
             <Image
-                className="object-cover w-full rounded-t-lg h-96 md:h-48 md:w-48 md:rounded-none md:rounded-l-lg"
+                className='transition-all ease-out duration-300 rounded-t-lg md:w-36 lg:w-44 xl:w-64 md:rounded-none md:rounded-l-lg'
```

Comment:
They best way i can figure out to fix this is to expand the width of the image.

Reply:
@Mosazghi Can you make the width dynamic so that the image keeps its ratio?